bitnami/node-exporter/README.md

```diff
@@ -0,0 +1,141 @@
+| `readinessProbe.timeoutSeconds`      | When the probe times out                                                                                 | `5`                                                                       |
+| `readinessProbe.failureThreshold`    | Minimum consecutive failures for the probe                                                               | `6`                                                                       |
+| `readinessProbe.successThreshold`    | Minimum consecutive successes for the probe                                                              | `1`                                                                       |
+| `serviceMonitor.enabled`             | Creates a ServiceMonitor to monitor Node Exporter                                                        | `false`                                                                   |
+| `serviceMonitor.namespace`           | Namespace in which Prometheus is running                                                                 | `nil`                                                                     |
+| `serviceMonitor.interval`            | Scrape interval (use by default, falling back to Prometheus' default)                                    | `nil`                                                                     |
+| `serviceMonitor.jobLabel`            | The name of the label on the target service to use as the job name in prometheus.                        | `nil`                                                                     |
+| `serviceMonitor.selector`            | ServiceMonitor selector labels                                                                           | `[]`                                                                      |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example the following command sets the `minReadySeconds` of the Node Exporter Pods to `120` seconds.
+
+```bash
+$ helm install --name my-release --set minReadySeconds=120 bitnami/node-exporter
+```
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```bash
+$ helm install --name my-release -f values.yaml bitnami/node-exporter
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+## Configuration and installation details
+
+### [Rolling VS Immutable tags](https://docs.bitnami.com/containers/how-to/understand-rolling-tags-containers/)
+
+It is strongly recommended to use immutable tags in a production environment. This ensures your deployment does not change automatically if the same tag is updated with a different image.
+
+Bitnami will release a new chart updating its containers if a new version of the main container, significant changes, or critical vulnerabilities exist.
+
+## Upgrading
+
+```bash
+$ helm upgrade my-release bitnami/node-exporter
+```
```

Comment:
Add a section describing the difference between **values.yaml** and **values-production.yaml** (check other charts)

Reply:
I am removing `values-production.yaml` as there really is no difference in the way the chart should be deployed for production env's